uPortal-portlets/src/main/java/org/apereo/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -115,18 +116,31 @@
 
     private static final String PORTLET_FNAME_FRAGMENT_ADMIN_PORTLET = "fragment-admin";
 
-    private enum PortletPermissionsOnForm {
+    /**
+     * Enumeration of the portlet-related permissions that may be managed in the Portlet Manager.
+     *
+     * @since 5.0
+     */
+    /* package-private */ enum PortletPermissionsOnForm {
         SUBSCRIBE(IPermission.PORTAL_SUBSCRIBE, IPermission.PORTLET_SUBSCRIBER_ACTIVITY),
         BROWSE(IPermission.PORTAL_SUBSCRIBE, IPermission.PORTLET_BROWSE_ACTIVITY),
         CONFIGURE(IPermission.PORTAL_PUBLISH, IPermission.PORTLET_MODE_CONFIG);
 
-        final String owner;
-        final String activity;
+        private final String owner;
+        private final String activity;
 
         PortletPermissionsOnForm(String owner, String activity) {
             this.owner = owner;
             this.activity = activity;
         }
+
+        public String getOwner() {
```

Comment:
I realize `owner` and `activity` were/are `final`, but they should have getter methods (otherwise they _look_ like a risky approach).